README.md

```diff
@@ -149,31 +149,33 @@ thereof) that is ready for use, with the following options:
 ````js
 {
   // Azure connection details for use with SAS from auth.taskcluster.net
-  account:           "...",              // Azure storage account name
-  table:             "AzureTableName",   // Azure table name
+  account:           "...",                    // Azure storage account name
+  table:             "AzureTableName",         // Azure table name
+  level:             "read-write || read-only" // Permissions for table
```

Comment:
Oh, adding this options is nice...
But we're probably better off making it a boolean `readOnly: true`, then the default is more obvious...

Also we probably should enforce it regardless of what kind of credentials is used. Ie. we should probably set ` subClass.prototype.__readOnly` and assert on it in the `.modify|.create|.remove|.ensureTable|.deleteTable` methods. Otherwise, we're likely to see someone use use `level: 'read-only'` and accidentally delete data.